### DISCUSSION TITLE
2.4.22 is out - import text description from members, support CBL

### DISCUSSION THREAD

**Discussion by edmundreinhardt** [IBM/ibmi-tobi#276] [Announcements]
A number of improvements.
2.4.21 supported COBOL
Support for older job log viewers like in Merlin 
`cvtsrcpf` now has a `-t` option to import the member description and store it as a comment at the top of the file.
BOB will read from these comments and use them on the TEXT() parm of the compile.

**Comment by NickLitten** [comment]
I prefer the 'OBJECT-this_is_description.something' format.

Is there a way to set this as default? I would like it to default when I click BUILD BOB from the Project Explorer. Any ideas?

**Comment by edmundreinhardt** [comment]
We did not remove the OBJ-description format support, simply use it.
